lib/Command/Lock.php

```diff
@@ -164,6 +167,12 @@ private function getStatus(InputInterface $input, OutputInterface $output, int $
 			$output->writeln(
 				'File #' . $fileId . ' is <comment>locked</comment> by ' . $lock->getUserId()
 			);
+			$output->writeln(
+				' - Locked at: ' . date('c', $lock->getCreation())
+			);
+			if ($lock->getETA() !== FileLock::ETA_INFINITE) {
+				$output->writeln(' - Expiry in seconds: ' . $lock->getETA());
```

Comment:
```suggestion
				$output->writeln(
					' - Expiry in seconds: ' . $lock->getETA()
				);
```
For consistency :grin:.